gems/pending/appliance_console/database_replication.rb

```diff
@@ -119,5 +116,14 @@ def run_repmgr_command(cmd, params = {})
       Process.wait(pid)
       $CHILD_STATUS.success?
     end
+
+    def primary_connection_hash
```

Comment:
I actually named it this way on purpose because using the attributes of this class you could be connecting to either the primary or secondary server. This is specifically the connection info for the primary server.

Or put another way, the name indicates what is returned, not who should be using the method.